src/pages/selectSalesOfferPackage.tsx

```diff
@@ -66,17 +65,24 @@ export interface SelectSalesOfferPackageProps {
     csrfToken: string;
 }
 
+export const formatSOPArray = (stringArray: string[]): string => {
+    let stringToReturn = '';
```

Comment:
This probably doesn't need to be exported if it's not being used elsewhere and we should be able to use a map and a join to simplify this a bit, so this could be:

`const formatSOPArray = (stringArray: string[]): string => stringArray.map(string => startCase(string)).join(', ')`